### PR TITLE
feat: add reassign reason + rename insights

### PR DESCRIPTION
**Pull request by sean-brydon**
## What does this PR do?

Here we add reassign reason to response csv export and also to data-table.

We also rename tabs to be summary,booking. Booking (previously routing) will provide a more detailed break down specific to bookings and routing form responses. Where as summary is/will become a more top level breakdown of the organization.

CleanShot 2024-11-18 at 11 36 01

## How should this be tested?
1) Reassign a routing form response booking with reason
2) Check table
3) Export data to CSV 
4) Open CSV - profit

## Checklist

- I haven't read the [contributing guide](https://github.com/calcom/cal.com/blob/main/CONTRIBUTING.md)
- My code doesn't follow the style guidelines of this project
- I haven't commented my code, particularly in hard-to-understand areas
- I haven't checked if my changes generate no new warnings

### CODE REVIEW

File: apps/web/pages/insights/bookings/index.tsx
Comment (sean-brydon) [inline apps/web/pages/insights/bookings/index.tsx:1]: Moved to /bookings > /routing

File: packages/features/insights/server/routing-events.ts [hunk 1]
```diff
@@ -238,6 +238,7 @@ class RoutingEventsInsights {
             uid: true,
             status: true,
             createdAt: true,
+            reassignReason: true,
```
Comment (sean-brydon) [inline packages/features/insights/server/routing-events.ts:241]: Add reassign reason to table

File: packages/features/shell/navigation/Navigation.tsx [hunk 2]
```diff
@@ -89,14 +89,14 @@ const navigation: NavigationItemType[] = [
     isCurrent: ({ pathname: path, item }) => path?.startsWith(item.href) ?? false,
     child: [
```
Comment (sean-brydon) [inline packages/features/shell/navigation/Navigation.tsx:90]: Rename tabs - see PR description for reasoning.

**Review by joeauyeung — CHANGES_REQUESTED** [review]
Heads up I've reassigned these two bookings but the reason isn't showing up. In `routing-events` I'm seeing the data contain the `routeToBooking` object.

image

**Pull request closed without merging**